src/test/ml/mlApi.spec.ts

```diff
@@ -28,6 +30,31 @@ const MODEL_1_RESPONSE = {
   },
 };
 
+const MODEL_2_RESPONSE = {
+  name: `projects/${PROJECT_ID}/models/${MODEL_ID_2}`,
+  createTime: "2020-02-07T22:45:23.288047Z",
+  updateTime: "2020-02-08T22:45:23.288047Z",
+  etag: "etag234",
+  modelHash: "modelHash234",
+  displayName: "model_2",
+  tags: ["tag_2", "tag_3"],
+  state: {},
+  tfliteModel: {
+    gcsTfliteUri: "gs://test-project-bucket/Firebase/ML/Models/model2.tflite",
+    sizeBytes: 26900988,
+  },
+};
+
+const MODEL_FILTER_STRING = "filter1";
+
+const MODEL_LIST_RESPONSE = {
```

Comment:
Seems like you use this type (object with models property which is an array of models) in a few places.  Maybe if you defined it as an `interface` the test wouldn't need `any` below?

Reply:
Oops. I did define the interface, I just forgot to use it here. Thanks!